lib/DDG/Spice/SimilarSites.pm

```diff
@@ -15,18 +16,22 @@ topics "everyday", "special_interest";
 attribution github => ['https://github.com/Adman', 'Adman'],
             twitter => ['http://twitter.com/adman_X', 'adman_X'];
 
-triggers startend => "similar sites", "similar sites to", "similar web to", "sites like", "websites like", "webs like", "pages like";
+triggers start => ["sites like", "site like", "websites like", "website like", "webs like", "pages like"];
+
+triggers any => ["similar sites", "similar web", "websites similar", "website similar", "similar websites", "similar website"];
```

Comment:
I figure we might as well support "similar to" as well. Seeing as we're checking to make sure we're working with a domain, it should be OK